doc/tech/approach.md

```diff
@@ -172,7 +174,7 @@ The logically centralized control plane is the foundation for the authoritative
 
 Although the control plane is logically centralized, scaling and control latency concerns must allow the actual implementation of the control plane to be distributed when required.
 
-All VNF, CNF and SW Virtualization acceleration or specific support functionality that could be programmed in the forwarding plane must be confined to the well separated sections or stages of any shared Underlay Networking. A practical example could be a SW Virtualization instance or VNF/CNF that control a NIC/SmartNIC where the Underlay Networking (Switch Fabric) ensures the separation in the same way as it is done for SR-IOV cases today.
+All VNF, CNF and SW Virtualization acceleration as well as all specific support functionality that could be programmed in the forwarding plane must be confined to the well separated sections or stages of any shared Underlay Networking. A practical example could be a SW Virtualization instance or VNF/CNF that control a NIC/SmartNIC where the Underlay Networking (Switch Fabric) ensures the separation in the same way as it is done for SR-IOV cases today.
```

Comment:
@TFredberg May I suggest a rewrite of the first sentence, "All VNF, CNF and SW Virtualization acceleration as well as all specific support functionality that is programmable in the forwarding plane must be confined to the well separated sections or stages of any shared Underlay Networking."

Second sentence, Typo "control" ---> "controls"

Reply:
Fixed